.storybook/main.js

```diff
@@ -1,4 +1,4 @@
-export default {
```

Comment:
When building Storybook, I get this warning, which goes away when the file is converted to a .js file.
```
info => Adding stories defined in "/Users/nyasskin/Documents/lucid/.storybook/main.js"
info => Loading custom manager config
WARN Unable to find main.js addons: /Users/nyasskin/Documents/lucid/.storybook/main
```

Reply:
I reverted this to main.ts -- still get the warning but storybook works.